src/stores/routePathStore.ts

```diff
@@ -90,11 +100,51 @@ export class RoutePathStore {
         return this._invalidLinkOrderNumbers;
     }
 
+    @computed
+    get kilpiViaNamesHash(): IKilpiViaHash {
```

Comment:
How about:
get kilpiViaName(relid): null | IKilpiViaName { }

There is no need to return the hash, other components doesn't have to know that about that kilpiVia's are stored in a hash